src/App.js

```diff
@@ -1,39 +1,101 @@
 import './App.css';
 import { useJson } from './hooks';
 import React, {useState} from "react"
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Remove this line since we using this here: https://github.com/SpectrrT/Pokedex-Elite-102/pull/4/files?diff=split&w=0#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R6